x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/layer_panel.tsx

```diff
@@ -351,11 +357,13 @@ export function LayerPanel(
                             aria-label={i18n.translate(
                               'xpack.lens.indexPattern.removeColumnLabel',
                               {
-                                defaultMessage: 'Remove configuration',
+                                defaultMessage: 'Remove configuration from {groupLabel}',
```

Comment:
Nit: This results in `Remove configuration from Vertical axis` which is a little weird regarding capitalization. Not sure whether it matters. An alternative would be `Remove configuration from "Vertical axis"`